webapi/src/it/scala/org/knora/webapi/ITTestDataFactory.scala

```diff
@@ -8,7 +8,7 @@ import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentif
 /**
  * Helps in creating value objects for tests.
  */
-object TestDataFactory {
+object ITTestDataFactory {
```

Comment:
```suggestion
object ItTestDataFactory {
```

Reply:
We once (before you were here) agreed on camelCasing acronyms / abbreviations, too.

Reply:
We are apparently not very consistent with this decision as there are all the `ADM` responders.
There is also an `ITKnoraLiveSpec` class.

If this is bothering us I suggest to create a followup issue and rename everything to this standard.
Since I have already a couple of branches open with this exact change in there it would lead to conflicts I am not ready to resolve just yet. 

In future I will take care of this naming convention once it is documented in some coding guidelines.